src/js/no-react-entry.js

```diff
@@ -28,7 +29,7 @@ createLoginWidget(store);
 if (pensionPages.has(location.pathname)) {
   createApplicationStatus(store, {
     formId: '21P-527EZ',
-    applyText: 'Apply for Veterans Pension'
+    applyText: 'Apply for Veterans Pension Benefits'
```

Comment:
Updated in response to this [feedback](https://github.com/department-of-veterans-affairs/vets-website/pull/6611/files#r145828134).